build_tools/azure/pylatest_pip_scipy_dev_linux-64_conda.lock

```diff
@@ -62,6 +62,6 @@ https://repo.anaconda.com/pkgs/main/linux-64/pip-23.3.1-py312h06a4308_0.conda#e1
 # pip meson-python @ https://files.pythonhosted.org/packages/91/c0/104cb6244c83fe6bc3886f144cc433db0c0c78efac5dc00e409a5a08c87d/meson_python-0.16.0-py3-none-any.whl#sha256=842dc9f5dc29e55fc769ff1b6fe328412fe6c870220fc321060a1d2d395e69e8
 # pip pooch @ https://files.pythonhosted.org/packages/f4/72/8ae0f1ba4ce6a4f6d4d01a60a9fdf690fde188c45c1872b0b4ddb0607ace/pooch-1.8.1-py3-none-any.whl#sha256=6b56611ac320c239faece1ac51a60b25796792599ce5c0b1bb87bf01df55e0a9
 # pip pytest-cov @ https://files.pythonhosted.org/packages/78/3a/af5b4fa5961d9a1e6237b530eb87dd04aea6eb83da09d2a4073d81b54ccf/pytest_cov-5.0.0-py3-none-any.whl#sha256=4f0764a1219df53214206bf1feea4633c3b558a2925c8b59f144f682861ce652
-# pip pytest-xdist @ https://files.pythonhosted.org/packages/e0/df/585742c204227526baf641e45a95d3c0a94978b85a1414622c1017ebfcf8/pytest_xdist-3.6.0-py3-none-any.whl#sha256=958e08f38472e1b3a83450d8d3e682e90fdbffee39a97dd0f27185a3bd9074d1
+# pip pytest-xdist @ https://files.pythonhosted.org/packages/6d/82/1d96bf03ee4c0fdc3c0cbe61470070e659ca78dc0086fb88b66c185e2449/pytest_xdist-3.6.1-py3-none-any.whl#sha256=9ed4adfb68a016610848639bb7e02c9352d5d9f03d04809919e2dafc3be4cca7
```

Comment:
The [recently released pytest-xdist 3.6.1 shipped the fix](https://github.com/pytest-dev/pytest-xdist/pull/1072#pullrequestreview-2026727241) for the race condition triggering the previous CI failure.